.github/workflows/preview-pr.yml

```diff
@@ -32,7 +25,6 @@ jobs:
       - name: Change Hugo baseurl so that PR preview works
         run: |
            sed -i 's#https://seismo-learn.org/#https://seismo-learn.org/sitepreview/seismo-learn/website/${{ github.head_ref }}/#' config/_default/config.toml
-        if: github.event_name == 'pull_request'
```

Comment:
This one is PR, removed?

Reply:
Yes, because the workflow only runs in `pull_request`, so no need to check the `github.event_name`.